conformance/packages/dns-test/src/fqdn.rs

```diff
@@ -4,7 +4,7 @@ use std::borrow::Cow;
 
 use crate::{Error, Result};
 
-#[derive(Clone, PartialEq)]
+#[derive(Clone, Eq, PartialEq, Ord, PartialOrd)]
```

Comment:
is `derive(PartialOrd)` the right thing to do here? for some reason, I had the impression that comparing FQDN would be done label by label and from right to left (TLD first; then go into the submdomains) rather than treating it as plain string but you know better the NSEC3 RFCs than I do :sweat_smile:

Reply:
you're right, the right comparison would be to do the comparison label from label and from right to left. However, this was a leftover of a previous iteration of the tests that I forgot to remove. I removed the extra derives